src/oncall/api/v0/ical_key.py

```diff
@@ -10,6 +10,27 @@ def generate_ical_key():
     return str(uuid.uuid4())
 
 
+def check_ical_team(team, requester):
+    """
+    Currently we allow users to request ical key for any team calendar
+    """
+    connection = db.connect()
+    cursor = connection.cursor()
+
+    cursor.execute(
+        '''
+        SELECT `id`
+        FROM `team`
+        WHERE `name` = %s AND `active` = TRUE
+        ''',
+        (team, ))
+    team_exist_and_active = cursor.rowcount
```

Comment:
Is it better to do `SELECT id` and then check for `cursor.rowcount` or is it better to do `select COUNT(*)` to get the count right away? 

I don't know enough about sql to definitely say...

Reply:
I think either works, and eventually we only want to know if `rowcount > 0`. I just followed elsewhere in the code on how to check for existence, like https://github.com/linkedin/oncall/blob/2ea0d7d12aa192d519bdc7e7a5ead4aa41cf809d/src/oncall/auth/__init__.py#L29